static/app/components/events/autofix/autofixFeedback.tsx

```diff
@@ -1,29 +1,33 @@
 import {useRef} from 'react';
 
 import {Button} from 'sentry/components/button';
-import useFeedbackWidget from 'sentry/components/feedback/widget/useFeedbackWidget';
 import {IconMegaphone} from 'sentry/icons/iconMegaphone';
 import {t} from 'sentry/locale';
+import {useFeedbackForm} from 'sentry/utils/useFeedbackForm';
 
 function AutofixFeedback() {
   const buttonRef = useRef<HTMLButtonElement>(null);
-  const feedback = useFeedbackWidget({
-    buttonRef,
-    messagePlaceholder: t('How can we make Autofix better for you?'),
-    optionOverrides: {
-      tags: {
-        ['feedback.source']: 'issue_details_ai_autofix',
-        ['feedback.owner']: 'ml-ai',
-      },
-    },
-  });
+  const openForm = useFeedbackForm();
```

Comment:
This uses the new `useFeedbackForm` hook which uses a form rendered at the root so it doesn't have the same issue as the other way